tests/integration/test_reshape.py

```diff
@@ -146,8 +149,11 @@ def test_1d(self):
             np.reshape(self.anp, (200,)),
         )
 
-    @pytest.mark.xfail
-    @pytest.mark.parametrize("order", ("C", "F", "A", "K", None), ids=str)
+    @pytest.mark.parametrize(
+        "order",
+        ("C", "F", "A", pytest.param("K", marks=pytest.mark.xfail), None),
```

Comment:
nice, I forgot that marks can be specified inline in parametrize lists. When this xfail issue is eventually fixed this arrangement will be simpler to update :+1: